### ISSUE TITLE
No matching version found for @storybook/client-logger@4.0.1

### ISSUE THREAD

**Issue by samosad** [storybooks/storybook#4660]
**Describe the bug**
Internal deps versions was not updated in 4.0.1 release

**To Reproduce**
1. install storybook versions like ^4.0.0
```
  "devDependencies": {
    "@storybook/addon-actions": "^4.0.0",
    "@storybook/addon-links": "^4.0.0",
    "@storybook/addon-options": "^4.0.0",
    "@storybook/addons": "^4.0.0",
    "@storybook/react": "^4.0.0",
```
2. See error in console:
```
Command failed: npm install 
npm ERR! code ETARGET
npm ERR! notarget No matching version found for @storybook/client-logger@4.0.1
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.
npm ERR! notarget
npm ERR! notarget It was specified as a dependency of '@storybook/core'
npm ERR! notarget
```

**Expected behavior**
Success install

**Comment by lehainam125** [comment]
+1. Same here.

**Comment by shilman** [comment]
Looking into it, thanks for letting me know!

**Comment by shaozj** [comment]
+1 I have to lock my version to 4.0.0

**Comment by shilman** [comment]
Looks like an NPM failure. Please try 4.0.2, which I just published and did a quick sanity check on.

**Comment by misolo** [comment]
@shilman I Have the same error with but **5.3.13** version

In package.json:
` "@storybook/addon-actions": "^5.3.13",`
    `"@storybook/addon-links": "^5.3.13",`
    `"@storybook/react": "^5.3.13"`

The error in console:
`npm ERR! notarget No matching version found for @storybook/node-logger@5.3.14`
`npm ERR! notarget In most cases you or one of your dependencies are requesting`
`npm ERR! notarget a package version that doesn't exist.`
`npm ERR! notarget `
`npm ERR! notarget It was specified as a dependency of '@storybook/react'`
`npm ERR! notarget `

**Comment by shilman** [comment]
@misolo looks like an NPM bug. can you clear your cache and try again?

https://www.npmjs.com/package/@storybook/node-logger

**Comment by misolo** [comment]
Resolved by upgrading @storybook/react to 5.3.14
